code/modules/projectiles/ammunition.dm

```diff
@@ -1,32 +1,38 @@
-//Magazine items, and casings.
-/*
-Boxes of ammo. Certain weapons have internal boxes of ammo that cannot be removed and function as part of the weapon.
-They're all essentially identical when it comes to getting the job done.
-*/
 /obj/item/ammo_magazine
 	name = "generic ammo"
 	desc = "A box of ammo."
 	icon = 'icons/obj/items/ammo.dmi'
 	icon_state = null
 	item_state = "ammo_mag" //PLACEHOLDER. This ensures the mag doesn't use the icon state instead.
-	var/bonus_overlay = null //Sprite pointer in ammo.dmi to an overlay to add to the gun, for extended mags, box mags, and so on
 	flags_atom = CONDUCT
 	flags_equip_slot = ITEM_SLOT_BELT
 	materials = list(/datum/material/metal = 100)
 	throwforce = 2
 	w_class = WEIGHT_CLASS_TINY
 	throw_speed = 2
 	throw_range = 6
-	var/datum/ammo/bullet/default_ammo = /datum/ammo/bullet/ //This is a typepath for the type of bullet the magazine holds, it is cast so that it can draw the variable handful_amount from default_ammo in create_handful()
-	var/datum/ammo/bullet/overcharge_ammo = /datum/ammo/bullet/ //Generally used for energy weapons
-	var/caliber = null // This is used for matching handfuls to each other or whatever the mag is. The #Defines can be found in __DEFINES/calibers.dm
-	var/current_rounds = -1 //Set this to something else for it not to start with different initial counts.
-	var/max_rounds = 7 //How many rounds can it hold?
-	var/gun_type = null //Path of the gun that it fits. Mags will fit any of the parent guns as well, so make sure you want this.
-	var/reload_delay = 1 //Set a timer for reloading mags. Higher is slower.
-	var/used_casings = 0 //Just an easier way to track how many shells to eject later.
-	var/flags_magazine = AMMUNITION_REFILLABLE //flags specifically for magazines.
-	var/base_mag_icon //the default mag icon state.
+	///Sprite pointer in ammo.dmi to an overlay to add to the gun, for extended mags, box mags, and so on
+	var/bonus_overlay = null
```

Comment:
tf you mean a pointer do you mean a ref?

Reply:
they mean the icon state